conjur/util/util_functions.py

```diff
@@ -27,3 +32,31 @@ def file_is_missing_or_empty(file):
     exists or the file size is zero; false otherwise
     """
     return not os.path.exists(file) or os.path.getsize(file) == 0
+
+def configure_env_var_with_keyring():
+    """
+    Setup the env variable for the desired keyring.
+    This is important so we will use the exact keyring method desired
+    and that 3rd party library will not choose other keyring option
```

Comment:
```suggestion
    Setup the ENV variable for the desired keyring.
    This is important so we will use the exact keyring backend required
    to successfully use the CLI and so that the Third party library will
    not favor another backend
```